File: corda-training-template/java-source/src/main/java/com/iou/java/api/IOUApi.java
package com.iou.java.api;

import com.iou.java.contract.IOUContract;
import com.iou.java.flow.IOUFlow;
import com.iou.java.state.IOUState;
import net.corda.core.contracts.ContractState;
import net.corda.core.contracts.StateAndRef;
import net.corda.core.crypto.Party;
import net.corda.core.messaging.CordaRPCOps;
import net.corda.core.transactions.SignedTransaction;

import javax.ws.rs.*;
import javax.ws.rs.core.MediaType;
import javax.ws.rs.core.Response;
import java.util.List;
import java.util.concurrent.ExecutionException;

// This API is accessible from /api/iou. All paths you specify are relative to this root.
@Path("iou")
public class IOUApi {
    private final CordaRPCOps services;

    public IOUApi(CordaRPCOps services) {
        this.services = services;
    }
}

File: corda-training-template/kotlin-source/src/main/kotlin/com/iou/kotlin/api/IOUApi.kt
package com.iou.kotlin.api

import com.iou.kotlin.contract.IOUContract
import com.iou.kotlin.flow.IOUFlow
import com.iou.kotlin.state.IOUState
import net.corda.core.messaging.CordaRPCOps
import java.util.concurrent.ExecutionException
import javax.ws.rs.*
import javax.ws.rs.core.MediaType
import javax.ws.rs.core.Response

// This API is accessible from /api/iou. All paths you specify are relative to this root.
@javax.ws.rs.Path("iou")
class IOUApi(val services: CordaRPCOps)